tests/rptest/e2e_tests/workloads/flink_table_transactions_basic.py

```diff
@@ -29,11 +30,25 @@
 MODE_CONSUME = 'consume'
 
 
+def shell(cmd) -> list[str]:
```

Comment:
nit: perhaps just use

```
check_output("find /opt/flink/opt/ -name flink-python*".split(), shell=True)
```

function seems out of place and only used once

Reply:
This script is running on flink environment and is completely isolated from ducktape. There can be only internals and pyflink dependencies. The proper way to handle this is to create configuration management for workloads that will handle the class libs searching and such. But that's a big and different task.

Reply:
didn't catch what you mean.. I was just saying we don't need this method in this file.